apps/git-pulse/src/routes/index.ts

```diff
@@ -0,0 +1,12 @@
+import SearchSuggestionsHandler from '../handlers/SearchSuggestionsHandler';
+import { Router } from 'express';
```

Comment:
There is a good practice rule for sorting imports. Please sort your imports like this:
1. Module imports (usually external libraries that you installed on the npm project) import { Router } from 'express';
2. Project imports (when we have any project module defined, like `import xxx from '@myapp/my-custom-file`
3. Relative imports (when you need to import any file based on the currnet path `import xxx from ./my/folder`